module/VuFind/src/VuFind/View/Helper/Root/OpenUrl.php

```diff
@@ -284,14 +284,7 @@ public function isActive()
         // check first if OpenURLs are enabled for this RecordDriver
         // check second if OpenURLs are enabled for this context
         // check last if any rules apply
-        if (
-            !$this->recordDriver->getOpenUrl()
-            || !$this->checkContext()
-            || !$this->checkIfRulesApply()
-        ) {
-            return false;
-        }
-        return true;
+        return !(!$this->recordDriver->getOpenUrl() || !$this->checkContext() || !$this->checkIfRulesApply());
```

Comment:
Can't this be simplified further?


```suggestion
        return $this->recordDriver->getOpenUrl() && $this->checkContext() && $this->checkIfRulesApply();
```